src/catalogue/items/catalogueItemsLandingPage.component.tsx

```diff
@@ -112,55 +148,203 @@ function CatalogueItemsLandingPage() {
       </Grid>
       {catalogueItemIdData && (
         <Grid item xs={12}>
-          <Box
-            sx={{
-              padding: '20px',
-              textAlign: 'left', // Left-align text
-              mx: '192px',
-            }}
-          >
-            <Typography
-              sx={{ margin: '8px', textAlign: 'center' }}
-              variant="h4"
-            >
-              {catalogueItemIdData.name}
-            </Typography>
-            <Typography sx={{ margin: '8px' }} variant="h6">
-              Description:
-            </Typography>
-            <Typography sx={{ margin: '8px' }} variant="body1">
-              {catalogueItemIdData.description}
-            </Typography>
-            <Box
-              onClick={toggleProperties}
-              style={{
-                display: 'flex',
-                flexDirection: 'row',
-                alignItems: 'center',
-                cursor: 'pointer',
-                margin: '8px',
-              }}
-              aria-label={`${
-                showProperties ? 'Close' : 'Show'
-              } catalogue item properties`}
-            >
-              {showProperties ? (
-                <>
-                  <Typography variant="h6">Properties</Typography>
-                  <ExpandLessIcon />
-                </>
-              ) : (
-                <>
-                  <Typography variant="h6">Properties</Typography>
-                  <ExpandMoreIcon />
-                </>
-              )}
-            </Box>
-            <Collapse in={showProperties}>
-              <Grid container spacing={1}>
-                {catalogueItemIdData.properties &&
-                  catalogueItemIdData.properties.map((property, index) => (
-                    <Grid item xs={12} sm={6} md={4} key={index}>
+          <Grid container spacing={1} flexDirection="column">
```

Comment:
Might be related to my generated data, but my landing page here has a strange horizonatal size causing a scrollbar when I think it should be able to fit
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/2d29a073-48bc-44c5-a1dd-91d1a9da8cb8)

Reply:
this is because of the manufacturer address. That can't be fix until matteo pr is in by adding line breaks  or by reordering the manufacturer details

Reply:
also description should be centered. I'll investigate that as well

Reply:
The issue was because I used margin instead of padding